package exception;

import java.util.InputMismatchException;
import java.util.Scanner;

public class ExceptionTest2 {

	public static void main(String[] args) {
		
		int num = 0;
		
		try {
			Scanner sc = new Scanner(System.in);
			System.out.print("양수를 입력하세요 >>");
			num = sc.nextInt();
		} catch (InputMismatchException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
			System.out.print("숫자 => 양수를 입력하시라니까요 !!!");
		}

	}

}
